fastplotlib/graphics/selectors/_linear.py

```diff
@@ -328,7 +338,7 @@ def _get_selected_index(self, graphic):
 
         if "Line" in graphic.__class__.__name__:
             # we want to find the index of the geometry position that is closest to the slider's geometry position
-            find_value = self.selection() - offset
+            find_value = self._selection() - offset
```

Comment:
once `_selection` is defined in the construction just access it using the property.